code/modules/mob/living/living_defense.dm

```diff
@@ -67,17 +67,15 @@
 		var/zone = ran_zone("chest", 65)//Hits a random part of the body, geared towards the chest
 		var/dtype = BRUTE
 		var/volume = I.get_volume_by_throwforce_and_or_w_class()
-		if(istype(I, /obj/item)) //If the item is a weapon...

```

Comment:
before someone reads this and goes "hurr durr" this used to say ```/obj/item/weapon``` but @Cyberboss 's regex killed it.